main/docs/fr-ca/customize/rules/create-rules.mdx

```diff
@@ -53,7 +53,7 @@ Si vous prévoyez d’utiliser des variables globales dans votre règle, veillez
 
 ## Utiliser le Dashboard
 
-1. Accédez à [Dashboard > Pipeline Auth > Règl](/docs/images///manage.auth0.com/#/rules) et cliquez sur **Créer**.
+1. Accédez à [Dashboard > Pipeline Auth > Règl](https://manage.auth0.com/#/rules) et cliquez sur **Créer**.
```

Comment:
🖐🏽 Change Request:
```suggestion
1. Accédez à [Dashboard > Auth Pipeline > Règl](https://manage.auth0.com/#/rules) et cliquez sur **Créer**.
```

🤔 Technically because it's French it should be "d’authentification", not "Auth".
<img width="254" height="112" alt="image" src="https://github.com/user-attachments/assets/9b7b17a6-2726-4396-91e9-164a799eb7ac" />